lib/bson/bson.js

```diff
@@ -383,6 +384,7 @@ module.exports.Symbol = Symbol;
 module.exports.BSON = BSON;
 module.exports.DBRef = DBRef;
 module.exports.Binary = Binary;
+module.exports.ObjectId = ObjectID;
```

Comment:
This seems like an unrelated change.

Reply:
It's related in removing confusing duplication in index.js. This is the one diffed thing not present.